locale/fr/LC_MESSAGES/messages.po

```diff
@@ -241,13 +240,12 @@ msgid "Axis labels"
 msgstr ""
 
 #: pygeoapi/templates/collections/coverage/domainset.html:18
-#, fuzzy
 msgid "Extent"
-msgstr "Suivant"
```

Comment:
Hey Joost!  It was partly my lack of knowledge in French and laziness that I made it blank.

Originally, gettext or Babel tool must have thought that "Next" and "Extent" look pretty close, so used "Suivant" as a suggested translation for "Extent", but marked it as _fuzzy_:

```
#: pygeoapi/templates/collections/coverage/domainset.html:18
#, fuzzy
msgid "Extent"
msgstr "Suivant"
```

As there seems "extent" could be "étendue", "ampleur", "portée" etc. in French (according to deepl.com), I wasn't sure which one is correct, and decided to be lazy and let this one go, but did remove the _fuzzy_ tag and clear the translation so there is no ambiguity because I know "Suivant" in French means "Next" and not "Extent".

(I got a bit carried away and spent too much time on updating the msgstr entries, when I hadn't started finished the more important PR #22 yet, so that was about the time I stopped with #23.  😅

Reply:
NP. Thx.